test/units/modules/network/ios/fixtures/ios_bgp_config.cfg

```diff
@@ -0,0 +1,11 @@
+router bgp 65536
+ bgp router-id 192.0.2.1
+ network 11.0.0.0 mask 255.255.0.0 route-map RMAP
```

Comment:
I don't want to touch test cases, but I worry that we have real IP addresses and as numbers here (not private ones)...